autogpt/llm/utils/__init__.py

```diff
@@ -109,14 +111,17 @@ def create_chat_completion(
         temperature = config.temperature
 
     logger.debug(
-        f"{Fore.GREEN}Creating chat completion with model {model}, temperature {temperature}, max_tokens {max_tokens}{Fore.RESET}"
+        f"{Fore.GREEN}Creating chat completion with model {model}, temperature {temperature}, max_tokens {max_tokens}, enable_function_calling: {config.enable_function_calling}, funcions: {functions}{Fore.RESET}"
```

Comment:
Dumping the functions here every cycle seems a bit much. Could be done at agent initialization instead.